### ISSUE TITLE
Only the first prop gets pushed to the wrapped component.

### ISSUE THREAD

**Issue by devinholloway** [ktsn/vuex-connect#44]
I have a route defined as:
    {
      path: '/batches/:batchId/books/:bookId',
      name: 'Book',
      component: BookContainer,
      props: true
   }

BookContainer is a vuex-connect component wrapping Book component.

Notice I have props:true, which should pass both batchId and bookId as props to BookContainer (which should pass them on to Book.

I've defined both props in Book:

  props: {
    batchId: {
      type: String,
      required: true
    },
    bookId: {
      type: String,
      required: true
    }
 }

I visit '/batches/1/books/2' and batchId is correctly set but bookId is always undefined.

**Comment by ktsn** [comment]
Could you provide reproduction?

**Comment by madarco** [comment]
Not sure if related, but the vuex-connect works only when used prop validation syntax, eg:
```
props: {
    propA: Number
}
```
And not with simple array:
```
props: [ 'propA' ]
```

Seems related to how `omit()` works.

To fix this add to createConnect (if you don't use it, create a `createConnect` function as described in the readme):
```
export default createConnect((options, lifecycle) => {
  if (Array.isArray(options.props)) {
    options.props = options.props.reduce((o, key) => ({ ...o, [key]: { type: null } }), {});
  }
  // ...
}
```

**Comment by ktsn** [comment]
@madarco It's not likely related with OP's problem but the array style props definition indeed should be supported too. I'll modify to support it in any cases.

**Comment by ktsn** [comment]
Implemented via #45.
I'm closing this issue since the reproduction for original report have not been provided for a long time. Please open a new issue if anyone follow up information. Thanks.
